### PR TITLE
Pbc/tax management

### PR DESCRIPTION
**Pull request by andriitserkovnyi**
## PR Description

Tax management PBC + includes

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by VadymSachenko — APPROVED** [review]

File: _data/sidebars/scos_dev_sidebar.yml [hunk 1]
```diff
@@ -162,8 +162,12 @@ entries:
       nested:
       - title: 'Reference information: Tax module'
```
Comment (jeremyfourna) [inline _data/sidebars/scos_dev_sidebar.yml:163]: All those tax pages in the dev section must also be moved to the PBC area

File: _data/sidebars/scos_user_sidebar.yml [hunk 2]
```diff
@@ -2722,14 +2722,6 @@ entries:
...
         nested:
-        - title: Create tax rates
-          url: /docs/scos/user/back-office-user-guides/administration/tax-rates/create-tax-rates.html
-          include_versions:
-          - "202204.0"
-        - title: Edit tax rates
-          url: /docs/scos/user/back-office-user-guides/administration/tax-rates/edit-tax-rates.html
-          include_versions:
-          - "202204.0"
         - title: Managing tax rates
```
Comment (jeremyfourna) [inline _data/sidebars/scos_user_sidebar.yml:2725]: Why is this part not removed?

File: _data/sidebars/scos_user_sidebar.yml [hunk 3]
```diff
@@ -2753,14 +2745,6 @@ entries:
...
         nested:
-        - title: Create tax sets
-          url: /docs/scos/user/back-office-user-guides/administration/tax-sets/create-tax-sets.html
-          include_versions:
-          - "202204.0"
-        - title: Edit tax sets
-          url: /docs/scos/user/back-office-user-guides/administration/tax-sets/edit-tax-sets.html
-          include_versions:
-          - "202204.0"
         - title: Managing tax sets
```
Comment (jeremyfourna) [inline _data/sidebars/scos_user_sidebar.yml:2748]: Why is this part not removed?

File: _data/sidebars/scos_user_sidebar.yml [hunk 4]
```diff
@@ -3196,13 +3180,11 @@ entries:
     - title: Taxes
```
Comment (jeremyfourna) [inline _data/sidebars/scos_user_sidebar.yml:3180]: Why is this part not removed?

File: _data/sidebars/scos_user_sidebar.yml [hunk 5]
```diff
@@ -3196,13 +3180,11 @@ entries:
...
       include_versions:
       - "202108.0"
-      - "202204.0"
       nested:
       - title: Avalara
```
Comment (jeremyfourna) [inline _data/sidebars/scos_user_sidebar.yml:3184]: Why is this part not removed?

File: _includes/pbc/all/data-import-and-export/file-details-product-abstract.csv.md [hunk 6]
```diff
@@ -48,16 +32,16 @@ The file should have the following parameters:
 | icecat_license |  | String | | Icecat product catalogue license code. |
 | new_from |  | Date | | To be considered a new product from this presented date. |
 | new_to |  | String | | To be considered a new product until this presented date. |
-| avalara_tax_code |  | String | | [Avalara tax code](/docs/scos/user/features/{{page.version}}/tax-feature-overview.html#avalara-system-for-automated-tax-compliance) for automated tax calculation. |
+| avalara_tax_code |  | String | | [Avalara tax code](/docs/pbc/all/tax-management/tax-feature-overview.html#avalara-system-for-automated-tax-compliance) for automated tax calculation. |
```
Comment (jeremyfourna) [inline _includes/pbc/all/data-import-and-export/file-details-product-abstract.csv.md:35]: We should only mention this part in the Avalara doc, not in the PIM default CSV import

File: docs/acp/user/apps/bazaarvoice.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
+---
```
Comment (jeremyfourna) [inline docs/acp/user/apps/bazaarvoice.md:1]: This file must be located in the PBC area, you can remove it

File: docs/acp/user/apps/payone.md [hunk 8]
```diff
@@ -4,10 +4,9 @@ description: Description
 template: howto-guide-template
```
Comment (jeremyfourna) [inline docs/acp/user/apps/payone.md:4]: This file must be located in the PBC area, you can remove it

File: docs/acp/user/apps/usercentrics.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
+---
```
Comment (jeremyfourna) [inline docs/acp/user/apps/usercentrics.md:1]: This file must be located in the PBC area, you can remove it

File: docs/pbc/all/payment-service-providers/psp.md [hunk 10]
```diff
@@ -1,21 +0,0 @@
----
```
Comment (jeremyfourna) [inline docs/pbc/all/payment-service-providers/psp.md:1]: Please rollback the file

File: docs/pbc/all/payment-service-providers/unzer/unzer.md [hunk 11]
```diff
@@ -1,22 +0,0 @@
----
```
Comment (jeremyfourna) [inline docs/pbc/all/payment-service-providers/unzer/unzer.md:1]: Please rollback the file

File: css/components/_sidebar-nav.scss [hunk 12]
```diff
@@ -70,7 +70,7 @@
         }
 
         .sidebar-nav__title {
-            font-weight: 500;
+            font-weight: normal;
```
Comment (jeremyfourna) [inline css/components/_sidebar-nav.scss:73]: You can rollback the change

File: _layouts/home.html [hunk 13]
```diff
@@ -45,7 +45,7 @@ <h2 class="card__heading-title">Developer guides</h2>
                             <li><a href="/docs/sdk/dev/spryker-sdk.html">Spryker SDK developer guides</a></li>
                             <li><a href="/docs/fes/dev/front-end-enablement-service.html">FeS developer guides</a></li>
                             <!-- <li><a href="/docs/paas-plus/dev/setup/system-requirements.html">Platform as a Service Plus developer guides</a></li> -->
-                            <li><a href="/docs/pbc/all/pbc.html">PBC guides</a></li>
+                            <li><a href="/docs/pbc/all/pbc-landing.html">PBC guides</a></li>
```
Comment (jeremyfourna) [inline _layouts/home.html:48]: You can rollback the change

**Review by jeremyfourna — CHANGES_REQUESTED** [review]
There are too many file changes in this PR to make a good review. Please roll back most of your change in the PBC area because the master already has a better structure around PSP and ratings and Reviews, same for ACP, where we moved the files around.
Ideally we should make multiple PR out of this ones.
1. Put all guides in the includes part PR
2. Tax PBC PR

File: _data/sidebars/pbc_all_sidebar.yml [hunk 14]
```diff
@@ -2,42 +2,74 @@ title: PBC guides
...
-      - title: Usercentrics
-        url: /docs/pbc/all/usercentrics.html
\ No newline at end of file
+                    url: /docs/pbc/all/payment/unzer/configuration-in-the-back-office/add-unzer-marketplace-credentials.html
+              - title: Customise the credit card display in your payment step
+                url: /docs/pbc/all/payment/unzer/customise-the-credit-card-display-in-your-payment-step.html
+              - title: Domain model & relationships
+                url: /docs/pbc/all/payment/unzer/domain-model-and-relationships.html
+      - title: Tax Management
+        url: /docs/pbc/all/tax-management/tax-feature-overview.html
```
[the remark this answers is not in the document]
Comment (andriitserkovnyi) [reply _data/sidebars/pbc_all_sidebar.yml:35]: Then, do you want to rewrite all feature overviews as capability overviews so we can use them as landing pages?

File: _data/sidebars/pbc_all_sidebar.yml [hunk 15]
```diff
@@ -2,42 +2,74 @@ title: PBC guides
...
+        nested:
+          - title: Manage in the Back Office
+            nested:
+              - title: Create tax rates
+                url: /docs/pbc/all/tax-management/manage-taxes-in-the-back-office/create-tax-rates.html
+              - title: Edit tax rates
+                url: /docs/pbc/all/tax-management/manage-taxes-in-the-back-office/edit-tax-rates.html
+              - title: Create tax sets
```
[the remark this answers is not in the document]
Comment (andriitserkovnyi) [reply _data/sidebars/pbc_all_sidebar.yml:43]: same

File: _data/sidebars/pbc_all_sidebar.yml [hunk 16]
```diff
@@ -2,42 +2,74 @@ title: PBC guides
...
+                url: /docs/pbc/all/tax-management/manage-taxes-in-the-back-office/create-tax-sets.html
+              - title: Edit tax sets
+                url: /docs/pbc/all/tax-management/manage-taxes-in-the-back-office/edit-tax-sets.html
+          - title: Manage via Glue API
```
[the remark this answers is not in the document]
Comment (andriitserkovnyi) [reply _data/sidebars/pbc_all_sidebar.yml:47]: Not sure what you mean

**Comment by andriitserkovnyi** [comment]
Split this PR

**Pull request closed without merging**